armi/operators/snapshots.py

```diff
@@ -26,11 +26,17 @@ class OperatorSnapshots(operatorMPI.OperatorMPI):
     This operator can be run as a restart, adding new physics to a previous run.
     """
 
-    def createInterfaces(self):
-        operatorMPI.OperatorMPI.createInterfaces(self)
+    def __init__(self, cs):
+        super().__init__(cs)
+
         # disable fuel management and optimization
         # disable depletion because we don't want to change number densities for tn's >0 (or any)
-        for toDisable in ["fuelHandler", "optimize", "depletion"]:
+        self.disabledInterfaces = ["depletion", "fuelHandler", "optimize"]
```

Comment:
you might consider a class variable. this is a singleton anyway....

Reply:
I considered it.  And that WOULD work, if you have strong opinions.

But, generally, we don't put things in class variables if we expect (or in this case _want_) people to edit them. 

It's certainly not a hard rule, so I'm open to the change. But people tend to use class variables for immutable data.